deploy-board/deploy_board/webapp/helpers/baseimages_helper.py

```diff
@@ -43,6 +49,18 @@ def get_all_with_acceptance(request, index, size):
                     'provider_name']] = img_info.get('acceptance') or 'UNKNOWN'
         img['acceptance'] = name_acceptance_map.get(img['provider_name'],
                                                     'N/A')
+
+        if name.startswith('cmp_base'):
+            if (name, cell) not in fetched_names_cells:
```

Comment:
What if (name, cell) was fetched, but the image isn't GOLDEN? 
cmp_base, us-east1, AMI-123
cmp_base, us-east1, AMI-345 <- GOLDEN

Reply:
yes, this is a bug, thanks